tests/end_to_end/global_decls/irix-o2-out.c

```diff
@@ -2,13 +2,17 @@ MIPS2C_UNK extern_fn(struct A *); // extern
 MIPS2C_UNK static_fn(struct A *); // static
 extern f32 extern_float;
 struct A *static_A_ptr = &static_A;
+s32 static_array[3] = {2, 4, 6};
+s32 static_bss_array[3] = {0, 0, 0};
```

Comment:
it would be nice to eliminate this initializer -- while GCC defaults to placing global variables explicitly initialized to zero in bss, IDO doesn't, and only places variables without explicit initializers there.

Reply:
Ooh nice insight. 
(This test didn't make that very obvious because the test is hand-edited to add the `rodata`/`data`/`bss` sections)